config/esp32/components/chip/Kconfig

```diff
@@ -1200,4 +1200,13 @@ menu "CHIP Device Layer"
 
     endmenu
 
+    menu "Enable BLE Extended Announcement"
+        config ENABLE_BLE_EXT_ANNOUNCEMENT
+            bool "Enable BLE Extended Announcement"
+            default n
+            help
+                Enable BLE Extended Announcement.
```

Comment:
Please update the help section as per our offline discussion

Reply:
done